themes/custom_tmpl/templates/search/home.phtml

```diff
@@ -0,0 +1,46 @@
+<?
```

Comment:
Copying the whole home.phtml file may cause long-term maintenance headaches -- I can easily see the core file changing and us forgetting to adjust this example. Is it possible to customize the template by rendering the parent version and wrapping things around it, or otherwise demonstrating customization with less duplication?